serving/docker/lmi.Dockerfile

```diff
@@ -123,6 +123,25 @@ RUN scripts/patch_oss_dlc.sh python \
     && pip3 cache purge \
     && apt-get clean -y && rm -rf /var/lib/apt/lists/*
 
+# Install OpenSSH for MPI to communicate between containers, allow OpenSSH to talk to containers without asking for confirmation
```

Comment:
could we wrap this whole code block as a shell script? Can potentially used in Neuron

Reply:
Added